app/models/manageiq/providers/openstack/cloud_manager/cloud_volume/operations.rb

```diff
@@ -18,6 +18,10 @@ def raw_attach_volume(server_ems_ref, device = nil)
         service.servers.get(server_ems_ref).attach_volume(ems_ref, device)
       end
     end
+  rescue => e
+    volume_name = name.empty? ? ems_ref : name
```

Comment:
Wouldn't be better use ```name.blank?``` to not fail when _name_ would be _nil_ or nil cannot appear there?

Reply:
I think you're right that `blank?` is the better choice here.

Reply:
Ended up using "presence" because rubocop complained :)